### PR TITLE
documentation: lilygo: fix formatting for LoRa32 and T8C3

### PR DESCRIPTION
**Pull request by lfelten**
Fix the formatting of the doc/index.rst for Lilygo LoRa32 and T8C3 Requested by @nordicjm

### CODE REVIEW

File: boards/lilygo/ttgo_t8c3/doc/index.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ enabled by moving a 0-ohm resistor.
 Connections and IOs
 ===================
 
-The TTGO T8-C3 board configuration supports the following hardware features:
+The ``ttgo_t8c3`` board target configuration supports the following hardware features:
```
Comment (kartben) [inline boards/lilygo/ttgo_t8c3/doc/index.rst:36]: Please only rename here, I.e. when directly referring to the Zephyr board target. All the other occurrences of the board name are fine to keep (and actually much more readable). 
Applies to the other file, and there is actually no need to change anything there AFAICS

File: boards/lilygo/ttgo_t8c3/doc/index.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .. _ttgo_t8c3:
 
-Lilygo TTGO T8-C3
-#################
+Lilygo TTGO T8C3
+################
```
Comment (kartben) [inline boards/lilygo/ttgo_t8c3/doc/index.rst:4]: Why this change? The commercial name of the board includes the hyphen afaict

File: boards/lilygo/ttgo_t8c3/doc/index.rst [hunk 3]
```diff
@@ -19,9 +19,9 @@ It features the following integrated components:
 
 .. figure:: img/ttgo_t8c3.webp
    :align: center
-   :alt: TTGO T8-C3
+   :alt: TTGO T8C3
```
Comment (kartben) [inline boards/lilygo/ttgo_t8c3/doc/index.rst:22]: Ditto

**Review by kartben — CHANGES_REQUESTED** [review]

File: boards/lilygo/ttgo_t8c3/doc/index.rst (hunk 1 above)
Comment (lfelten) [reply boards/lilygo/ttgo_t8c3/doc/index.rst:36]: Ok, fixed.

File: boards/lilygo/ttgo_t8c3/doc/index.rst (hunk 3 above)
Comment (lfelten) [reply boards/lilygo/ttgo_t8c3/doc/index.rst]: Reverted the T8C3 changes

File: boards/lilygo/ttgo_t8c3/doc/index.rst (hunk 2 above)
Comment (lfelten) [reply boards/lilygo/ttgo_t8c3/doc/index.rst]: I was unsure if it should reflect the board name or the target config name. Naming for these products is not consistent: T8C3, T8-C3 and T8_C3 are all used.

**Review by kartben — DISMISSED** [review]

File: boards/lilygo/ttgo_lora32/doc/index.rst [hunk 4]
```diff
@@ -178,7 +178,7 @@ Build and flash applications as usual (see :ref:`build_an_application` and
    :board: ttgo_lora32/esp32/procpu
    :goals: build
 
-The usual ``flash`` target will work with the ``ttgo_lora32`` board
+The usual ``flash`` target will work with the ``ttgo_lora32`` board target
 configuration. Here is an example for the :zephyr:code-sample:`hello_world`
```
Comment (nordicjm) [inline boards/lilygo/ttgo_lora32/doc/index.rst:182]: -configuration

File: boards/lilygo/ttgo_t8c3/doc/index.rst (hunk 1 above)
Comment (nordicjm) [inline boards/lilygo/ttgo_t8c3/doc/index.rst:36]: as above

File: boards/lilygo/ttgo_t8c3/doc/index.rst [hunk 5]
```diff
@@ -176,7 +176,7 @@ Build and flash applications as usual (see :ref:`build_an_application` and
    :board: ttgo_t8c3
    :goals: build
 
-The usual ``flash`` target will work with the ``ttgo_t8c3`` board
+The usual ``flash`` target will work with the ``ttgo_t8c3`` board target
 configuration. Here is an example for the :zephyr:code-sample:`hello_world`
```
Comment (nordicjm) [inline boards/lilygo/ttgo_t8c3/doc/index.rst:180]: as above

File: boards/lilygo/ttgo_lora32/doc/index.rst (hunk 4 above)
Comment (lfelten) [reply boards/lilygo/ttgo_lora32/doc/index.rst]: Sorry, I'm unsure when to use target or target configuration.
Fixed.

File: boards/lilygo/ttgo_t8c3/doc/index.rst (hunk 1 above)
Comment (lfelten) [reply boards/lilygo/ttgo_t8c3/doc/index.rst]: dito

File: boards/lilygo/ttgo_t8c3/doc/index.rst (hunk 5 above)
Comment (lfelten) [reply boards/lilygo/ttgo_t8c3/doc/index.rst]: dito

**Review by kartben — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]
